addons/binding/org.openhab.binding.sensebox/src/main/java/org/openhab/binding/sensebox/internal/handler/SenseBoxHandler.java

```diff
@@ -243,7 +251,49 @@ private State decimalFromSensor(SenseBoxSensor data) {
         if (data != null) {
             if (data.getLastMeasurement() != null) {
                 if (StringUtils.isNotEmpty(data.getLastMeasurement().getValue())) {
-                    result = new DecimalType(data.getLastMeasurement().getValue());
+                    logger.debug("About to determine quantity for {} / {}", data.getLastMeasurement().getValue(), data.getUnit());
+                    BigDecimal bd = new BigDecimal(data.getLastMeasurement().getValue());
+
+                    try {
+                        switch (data.getUnit()) {
+                            case "%":
+                                result = new QuantityType<>(bd, SmartHomeUnits.ONE);
+                                break;
+                            case "°C":
+                                result = new QuantityType<>(bd, SIUnits.CELSIUS);
+                                break;
+                            case "Pa":
+                                result = new QuantityType<>(bd, SIUnits.PASCAL);
+                                break;
+                            case "hPa":
+                                if (BigDecimal.valueOf(10000l).compareTo(bd) < 0) {
+                                    // Some stations report measurements in Pascal, but send 'hPa' as units...
+                                    bd = bd.divide(ONEHUNDRED);
+                                }
+                                result = new QuantityType<>(bd, HECTO(SIUnits.PASCAL));
+                                break;
+                            case "lx":
+                                result = new QuantityType<>(bd, SmartHomeUnits.LUX);
+                                break;
+                            case "µg/m³":
+                                // result = new QuantityType<>(bd, SmartHomeUnits.MICROGRAM_PER_CUBICMETRE);
+                                result = new QuantityType<>(bd, SmartHomeUnits.KILOGRAM_PER_CUBICMETRE);
+                                break;
+                            case "μW/cm²":
+                                // result = new QuantityType<>(bd, SmartHomeUnits.MICROWATT_PER_SQUARE_CENTIMETRE);
+                                result = new QuantityType<>(bd, SmartHomeUnits.IRRADIANCE);
+                                break;
+                            default:
+                                logger.info("    Could not determine unit for '{}', using default", data.getUnit());
+                                result = new QuantityType<>(bd, SmartHomeUnits.ONE);
+                        }
+                    } catch (java.lang.IllegalStateException ise) {
+                        // java.lang.IllegalStateException: Error invoking #valueOf(String) on class 'org.eclipse.smarthome.core.library.types.QuantityType' with value '0.00 μg/m³'.
+                        logger.error("Could not create measured quantity", ise);
```

Comment:
If this is not a severe bug it should be logged on warn, see also the [Logging Guidelines](https://www.openhab.org/docs/developer/development/guidelines.html#e-logging):

> `error` logging should only be used to inform the user that something is tremendously wrong in his setup, the system cannot function normally anymore, and there is a need for immediate action. It should also be used if some code fails irrecoverably and the user should report it as a severe bug.

Reply:
Done.